ndk/src/events/encryption.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * Encryption and gift-wrapping of events
+ * Implements Nip04, Nip44, Nip59 
+ */
+import type { NDKSigner } from "../signers";
+import { NDKUser } from "../user";
+import { NDKEvent } from "./index.js";
+
+// NIP04 && NIP44
+
+export type EncryptionNip = 'nip04' | 'nip44';
+export type EncryptionMethod = 'encrypt' | 'decrypt'
+
+// some clients may wish to set a default for message encryption...
+let defaultEncryption : EncryptionNip | undefined = undefined;
+export function useEncryption(nip : EncryptionNip) {
```

Comment:
nitpick: the extra space between the param name and the `:`

Reply:
I've noticed this in quite a few spots.

Reply:
As I wrote above, I'd still format the PR files with the project's prettier config.

Reply:
@erskingardner please tell me whether I should format the changes, after that reviews are more cumbersome.

Reply:
You know – let's just leave that. It can be done in another PR that is just a formatting PR.

Reply:
Is it ready for a final review now?

Reply:
Yes.